au4/src/project/view/projectpropertiesmodel.cpp

```diff
@@ -36,13 +36,22 @@ ProjectPropertiesModel::ProjectPropertiesModel(QObject* parent)
     //     m_projectMetaInfo = project->metaInfo();
     // }
 
+}
+
+void ProjectPropertiesModel::init()
+{
+    if (!m_project) {
```

Comment:
`m_project = context()->currentProject();` - it's better to call it in init.
Also we should subscribe to project changes in global context

Reply:
fixed